cassis/cas.py

```diff
@@ -591,7 +678,10 @@ def _find_all_fs(self) -> Iterable[FeatureStructure]:
                         openlist.append(referenced_fs)
 
         # We do not want to return cas:NULL here as we handle serializing it later
-        all_fs.pop(0, None)
+        for fs_id, fs in list(all_fs.items()):
```

Comment:
Why do you convert it to a list?

Reply:
because I am modifying the dictionary as I iterate over it and don't want to get any concurrent modification problems.